guardrails/src/use_cases/create_config.py

```
@@ -0,0 +1,69 @@
+        Raises:
+            ConfigAlreadyExistsError: If a config with the same name exists
+            InvalidConfigError: If the configuration content is invalid
+        """
+        # Check for duplicate name
+        existing = await self.repo.get_config_by_name(name)
+        if existing:
+            raise ConfigAlreadyExistsError(name)
+
+        # Validate config content
+        if not config_yaml.strip():
+            raise InvalidConfigError("config_yaml cannot be empty")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)